humanguide/design-concepts/app-icon-design.rst

```diff
@@ -0,0 +1,109 @@
+App icon design
+===============
+
+The whole system and apps follow the `Suru design principles <http://web.archive.org/web/20191223082720/https://docs.ubuntu.com/phone/en/apps/design/get-started/design-philosophy>`__ started by Canonical and `Samuel Hewitt <https://icons.samuelhewitt.com/>`__.
+
+Design Philosophy and Suru Meaning
+----------------------------------
+
+-  Suru alludes to Japanese culture.
+-  The design of Suru is inspired by origami.
+-  Paper can be used in two and three dimensional forms, because it is
+   transferable and diverse.
+
+Suru brings a precise yet organic structure to the Ubuntu Touch interface. The sharp lines and varying levels of transparency evoke the edges and texture of paper.
+
+Design Tips for an App Icon
+---------------------------
+
+Descriptive design
+~~~~~~~~~~~~~~~~~~
+
+The icon you choose for your app should describe what your app is about, what it does, or what service it is there for in a recognizable way.
+
+.. figure:: /_static/images/humanguide/03.png
```

Comment:
Might we want to show these icons at various sizes in a single image? This will improve the examples saying that the icons should be descriptive even at small sizes.